doc/build-unix-ubuntu.md

```diff
@@ -0,0 +1,51 @@
+eBoost Node - UBUNTU 16
+
+Instructions for running an eBoost (EBST) node
+Run steps 1-4 with sudo account.  Create new user account to run node. Best practice is to run node with basic user with no sudo access.
+
+Recommended system configuration: 4GB RAM, 2 CPU, 20GB storage
+Disk space usage: 2.2GB
+
+Source code location:  https://github.com/EBoost-dev/EBoost-dev
+
+1.	Start with Ubuntu 16	Commands below  ( You may also use CentOS or Ubuntu 18, see separate build instructions)
+2.	Update linux	
+    > sudo apt-get update
+3.	Install Dependencies	
+    > sudo apt-get install build-essential libtool autotools-dev automake pkg-config libssl-dev libevent-dev bsdmainutils ssh libboost-all-dev
+4.	Update linux	
+    > sudo apt-get update
+5.	Github clone	
+    > git clone https://github.com/EBoost-dev/EBoost-dev eboost
+6.	Change dir	
+    > cd eboost
+7.	Fetch params	
+    > ./autogen.sh
+8.	Build	
+    > ./configure
+9.	Make PTX directory	
```

Comment:
Remove references to PTX plz ;) I assume this is here from ripping the build guide from PTX coin. Its mentioned a couple times.

Reply:
lol ok.  will fix